src/cgame/tr_types.h

```diff
@@ -32,6 +32,7 @@ If you have questions concerning this license or the applicable additional terms
 
 #define MAX_CORONAS     32          //----(SA)	not really a reason to limit this other than trying to keep a reasonable count
 #define MAX_DLIGHTS     32          // can't be increased, because bit flags are used on surfaces
+#define MAX_ENTITIES    1023        // can't be increased without changing drawsurf bit packing
```

Comment:
Looks like cg_localents uses it in ET, hmm.  Could also be replaced with ENTITYNUM_MAX_NORMAL, maybe? Can change later.